### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by WardenGnaw**
Update page to match
https://github.com/microsoft/repo-templates/blob/main/shared/CODE_OF_CONDUCT.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
-## Microsoft Open Source Code of Conduct
+# Microsoft Open Source Code of Conduct
```
Comment (bobbrow) [inline CODE_OF_CONDUCT.md:1]: That's too big of a heading in my opinion.  Let's keep it as `##`

Comment (bobbrow) [reply CODE_OF_CONDUCT.md:1]: I missed the part that this is a separate file from the readme.md.  `#` is fine here.  Is there any guidance about updating the readme?  These two sections used to have the same content but I see the readme of the template repo does not match this either.

Comment (WardenGnaw) [reply CODE_OF_CONDUCT.md:1]: The template use to exist at a different page, most likely at https://opensource.microsoft.com/. 

Now they updated it to exist at this https://github.com/microsoft/repo-templates/blob/main/shared/CODE_OF_CONDUCT.md.

Comment (WardenGnaw) [reply CODE_OF_CONDUCT.md:1]: Oh. I see you mean this from the README.md

https://github.com/microsoft/repo-templates/blob/main/README.md#code-of-conduct

Comment (WardenGnaw) [reply CODE_OF_CONDUCT.md:1]: Yes. There is no guidance on updating the README atm.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**